rust_src/crates/webrender/src/output.rs

```diff
@@ -359,13 +361,24 @@ impl Output {
 
         let font_index = font_handle.index;
         match font_handle.source.as_ref() {
+            #[cfg(all(unix, not(target_os = "macos")))]
             Source::File(path) => {
                 let font = NativeFontHandle {
                     path: path.clone().into_os_string().into(),
                     index: font_index,
                 };
                 txn.add_native_font(font_key, font);
             }
+            #[cfg(target_os = "macos")]
+            Source::File(path) => {
+                let font = NativeFontHandle(
```

Comment:
I will create a new PR into emacs-ng to help avoiding this.

Reply:
Oh, Do you mean the hard coding? Right, I did that to make macOS build pass.
Can we have platform specific code abstract layout like [servo](https://github.com/servo/servo/tree/master/components/gfx/platform)

Reply:
https://github.com/emacs-ng/emacs-ng/pull/375